lib/buildFactory.js

```diff
@@ -268,10 +268,16 @@ class BuildFactory extends BaseFactory {
 
                     const build = await super.create(modelConfig);
 
-                    await steps.reduce(
-                        (p, step) => p.then(() => stepFactory.create({ buildId: build.id, ...step })),
-                        Promise.resolve()
-                    );
+                    // eslint-disable-next-line no-restricted-syntax
+                    for (const step of steps) {
+                        try {
+                            // eslint-disable-next-line no-await-in-loop
+                            await stepFactory.create({ buildId: build.id, ...step });
+                        } catch (err) {
+                            // eslint-disable-next-line max-len
+                            throw new Error(`Error in steps -BuildId:${build.id}-step:${step}`, err);
```

Comment:
I think it is also better to log these message using `logger.error`.

Reply:
ok. I modify to using `logger`